src/requirements/data/minors/ineq.ts

```diff
@@ -0,0 +1,235 @@
+import { CollegeOrMajorRequirement } from '../../types';
+import { includesWithSubRequirements } from '../checkers-common';
+
+const inequalityRequirements: readonly CollegeOrMajorRequirement[] = [
+  {
+    name: 'Inequality Core Course',
+    description:
+      'This course introduces students to contemporary debates and controversies' +
+      ' about inequality. Learn how and why inequality varies across time and space,' +
+      ' and learn to identify how inequality is generated and maintained. Offered every fall semester.',
+    source: 'https://inequality.cornell.edu/minor-overview-and-requirements',
+    checker: includesWithSubRequirements([
+      'SOC 2220',
+      'DSOC 2220',
+      'PAM 2220',
+      'ILROB 2220',
+      'PHIL 1950',
+      'AMST 2225',
+      'GOVT 2225',
+    ]),
+    fulfilledBy: 'courses',
+    perSlotMinCount: [1],
+    slotNames: ['Course'],
+  },
+  {
+    name: 'Inequality Overview Course',
+    description: 'An Overview Course',
+    source: 'https://inequality.cornell.edu/minor-overview-and-requirements',
+    checker: includesWithSubRequirements([
```

Comment:
I don't see PAM 2208 or DSOC 2090 as an option for overview courses. Besides those, this looks ok.